Dockerfile

```diff
@@ -7,8 +7,8 @@ FROM python:bookworm
 LABEL maintainer="rht <rhtbot@protonmail.com>"
```

Comment:
`LABEL maintainer="projectmesa <maintainers@projectmesa.dev>"

Reply:
Modified accroding to the suggestions